extensions/wikia/Paginator/tests/PaginatorTest.php

```diff
@@ -194,7 +194,7 @@ public function setUp() {
 		$this->setupFile = "$IP/extensions/wikia/Paginator/Paginator.setup.php";
```

Comment:
consider using $this->setupFile = dirname( __FILE__ ) . '/../Paginator.setup.php'; ?